.poshchan/settings.json

```diff
@@ -11,16 +16,20 @@
             "PowerShell-CI-Linux"
         ]
     },
-    "authorized_users": [
-        "adityapatwardhan",
-        "anmenaga",
-        "daxian-dbw",
-        "iSazonov",
-        "JamesWTruher",
-        "PaulHigin",
-        "rjmholt",
-        "SteveL-MSFT",
-        "TravisEz13",
-        "TylerLeonhardt"
-    ]
+    "authorized_users": {
+        "build_targets": [
+            "adityapatwardhan",
+            "anmenaga",
+            "daxian-dbw",
+            "iSazonov",
+            "JamesWTruher",
+            "PaulHigin",
+            "rjmholt",
+            "SteveL-MSFT",
+            "TravisEz13",
+            "TylerLeonhardt",
+            "markekraus"
+        ],
+        "reminders": "*"
```

Comment:
I don't think we are ready for everyone to test this.  This is still a preview.  @SteveL-MSFT 	  Are you ready to open this to a few people?

Reply:
I'm ok with reminders being open to everyone.  That part of the code is pretty trivial and doesn't use much resources.